editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -66,12 +172,168 @@ export const ClassNameSelect: React.FunctionComponent = betterReactMemo('ClassNa
+        display: 'flex',
+        alignItems: 'center',
+        gap: 4,
+        paddingLeft: 4,
+        paddingRight: 4,
+        paddingTop: 0,
+        paddingBottom: 0,
+        maxWidth: 0,
+      }),
+
+      multiValue: () => {
+        return {
+          cursor: 'pointer',
+          display: 'flex',
+          alignItems: 'center',
+          height: 18,
+          backgroundColor: '#191818',
+        }
+      },
+      multiValueLabel: () => ({
+        fontSize: 10,
+        padding: '2px 4px',
+        color: 'white',
+      }),
+      multiValueRemove: (styles: React.CSSProperties, { data }) => ({
+        width: 11,
+        display: 'flex',
+        paddingTop: 2,
+        opacity: 0.4,
+        color: data.color,
+        ':hover': {
+          opacity: 1,
+          backgroundColor: data.color,
+          color: 'white',
+        },
+      }),
+      input: () => {
+        return {
+          fontSize: 11,
+          color: theme.fg1.value,
+          letterSpacing: 0.3,
+          background: 'transparent',
+          display: 'flex',
+          alignItems: 'center',
+        }
+      },
+      indicatorsContainer: (styles) => ({
+        ...styles,
+        height: 20,
+      }),
+      option: (styles: React.CSSProperties, { data, isDisabled, isFocused, isSelected }) => {
+        // a single entry in the options list
+        const categories = data?.categories ?? []
+
+        let optionColor = ChromaBlack
+        if (categories.length === 1) {
+          optionColor = ChromaThemePrimary
+        }
+
+        let backgroundColor: string | undefined = undefined
+        if (isFocused) {
+          backgroundColor = optionColor.alpha(0.1).css()
+        } else if (isSelected) {
+          backgroundColor = optionColor.css()
+        } else if (isDisabled) {
+          backgroundColor = undefined
+        }
+
+        let activeBackgroundColor =
+          !isDisabled && (isSelected ? data.color : optionColor.alpha(0.3).css())
+
+        let color: string = optionColor.css()
+        if (isSelected) {
+          if (chroma.contrast(optionColor, 'white') > 2) {
+            color = 'white'
+          } else {
+            color = 'black'
+          }
+        } else if (isDisabled) {
+          color = '#ccc'
+        }
+
+        return {
+          minHeight: 27,
+          display: 'flex',
+          alignItems: 'center',
+          paddingLeft: 8,
+          paddingRight: 8,
+          backgroundColor: backgroundColor,
+          color: color,
+          cursor: isDisabled ? 'not-allowed' : 'default',
+
+          ':active': {
+            ...(styles as any)[':active'],
+            backgroundColor: activeBackgroundColor,
+          },
+        }
+      },
+    }),
+    [theme, ChromaThemePrimary],
+  )
+
+  const filterOption = React.useCallback((option: Option, rawInput: string) => {
+    return createFilter({ ignoreAccents: false })(option, rawInput)
+  }, [])
```

Comment:
Ahhh I misunderstood what was happening here. In this case, the best option is to have a `const filterOption = createFilter({ ignoreAccents: false })` at the root level of the file